src/core/utils/matching/matchRequestUrl.ts

```diff
@@ -55,8 +55,8 @@ export function coercePath(path: string): string {
 /**
  * Returns the result of matching given request URL against a mask.
  */
-export function matchRequestUrl(url: URL, path: Path, baseUrl?: string): Match {
-  const normalizedPath = normalizePath(path, baseUrl)
+export function matchRequestUrl(url: URL, path: Path): Match {
```

Comment:
This is a public API change. We have to keep it for any consumers depending on this function. Perhaps it's even us.